pkg/agent/baker_test.go

```diff
@@ -532,8 +532,19 @@ var _ = Describe("Assert generated customData and cseCmd", func() {
 			config.KubeletConfig = map[string]string{}
 			config.AgentPoolProfile.VMSize = "Standard_ND96asr_v4"
 			config.EnableNvidia = true
-			config.GPUInstanceProfile = "mig-3g"
+			config.GPUInstanceProfile = "MIG7g"
```

Comment:
Why is this being changed from "mig-3g" to "MIG7g", in this test (for the same VM size)? Just to test the 7 partitions?

Reply:
mig-3g is invalid syntax, figured it's better to use appropriate profiles.

MIG3g is a valid profile. 1,2,3,4,7 are all valid currently valid. see https://docs.nvidia.com/datacenter/tesla/mig-user-guide/index.html#a100-profiles